mypyc/test-data/analysis.test

```diff
@@ -37,7 +37,7 @@ L4:
 L5:
     z = 2
 L6:
-    r5 = None
+    r5 = 1
     return r5
```

Comment:
Could this be simplified to `return 1`, similar to how we simplify constant value returns in other contexts?